user.go

```diff
@@ -96,6 +96,15 @@ type user struct {
 
 	networks        []*network
 	downstreamConns []*downstreamConn
+
+	pendingListsLock sync.Mutex
+	pendingLists     []pendingList
+}
+
+type pendingList struct {
+	downstreamID uint64
+	// list of pending per-upstream LIST requests and their corresponding LIST message parameters
```

Comment:
```go
// list of per-upstream LIST commands not yet sent
```

Reply:
`not yet complete`*? we only clear it when we receive the last RPL_LISTEND (or error, or close)

Reply:
Eh, right! +1